Assets/Scripts/Gameplay/Level1/ChurchEvent.cs

```diff
@@ -1,28 +1,31 @@
 using UnityEngine;
-using Cinemachine;
 
 namespace ProjectWendigo
 {
     public class ChurchEvent : MonoBehaviour
     {
+        [SerializeField] public float Duration = 3f;
+
         private void Awake()
         {
             Singletons.Main.Event.On("ChurchEvent", this.EnterEvent);
         }
 
+        void Start()
+        {
+            Singletons.Main.Event.Trigger("ChurchEvent");
+        }
+
         public void EnterEvent()
         {
             this.ChangeSensitivity();
+            this.GetComponent<NearingBreathing>().Trigger(this.Duration);
+            this.Invoke(nameof(this.ChangeSensitivity), this.Duration, 0.2f);
```

Comment:
Use `Singletons.Main.Camera.PlayerCamera.GetComponent<CinemachineVirtualCamera>().GetCinemachineComponent<CinemachinePOV>().m_VerticalAxis.m_MaxSpeed` instead of `0.2f`, so that we can change sensitivity without this breaking.